src/cli/cli.cpp

```diff
@@ -8319,6 +8319,34 @@ template <> otError Interpreter::Process<Cmd("wakeup")>(Arg aArgs[])
         error = ProcessEnableDisable(aArgs + 1, otLinkIsWakeupListenEnabled, otLinkSetWakeUpListenEnabled);
     }
 #endif // OPENTHREAD_CONFIG_WAKEUP_END_DEVICE_ENABLE
+#if OPENTHREAD_CONFIG_WAKEUP_COORDINATOR_ENABLE
+    /**
+     * @cli wakeup wake
+     * @code
+     * wakeup wake 1ece0a6c4653a7c1 7500 1090
+     * Done
+     * @endcode
+     * @cparam Wake-up End Device extended address @ca{extaddr}
+     * @cparam wake-up interval (us) @ca{wakeup-interval}
+     * @cparam wake-up duration (ms) @ca{wakeup-duration}
```

Comment:
The cparam tag should be a single line with the syntax of all parameters.  So like

`@cparam wakeup wake @ca{extaddr} @ca{wakeup-interval} @ca{wakeup-duration}`

https://openthread.io/reference/cli/contribute#parameters

If you need to add more detail you can do that in a `@par` after